### PR TITLE
feat: export plugins

### PR DESCRIPTION
**Pull request by astandrik**
Adds plugins export for manual plugin registration in project

### CODE REVIEW

**Comment by jhoncool** [comment]
@astandrik, Hi! I think it's a bad idea to export types `PluginObjectSettings`, `PluginObject` inside `Text` plugin to a shared index.ts. 

If you still want to do this, then let's change the name of the type: `PluginObjectSettings` -> `PluginTextObjectSettings`, `PluginObject` -> `PluginTextObject`. 

And someday in a major version I will change the name in the file `Text.tsx`

**Review by artemluchin — APPROVED** [review]

**Comment by astandrik** [comment]
@jhoncool yeah you right
fixed

**Comment by astandrik** [comment]
@jhoncool 
commited fixes in Readme and exports
check again plz

File: src/plugins/Text/index.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+export {
+    PluginObjectSettings as PluginTextObjectSettings,
```
Comment (jhoncool) [inline src/plugins/Text/index.ts:2]: Can you export also `PluginTextApiHandler` and `PluginTextProps`?

**Review by jhoncool — APPROVED** [review]

**Pull request merged**